include/mxnet/executor.h

```diff
@@ -152,14 +152,14 @@ class Executor {
   static Executor* SimpleBind(nnvm::Symbol symbol,
```

Comment:
@piiswrong I think we already discussed this [here](https://github.com/apache/incubator-mxnet/pull/11325/files/6ce43cb90658e8cb13177a0210093b567e33e4f6#r197672536). The TensorRT graph pass does a graph rewrite, and that requires mutable types. Going from const ref to ref causes the linter to fail. The linter suggestion is to switch from const refs to pointer if mutability is desired. You mentioned [here](https://github.com/apache/incubator-mxnet/pull/11325/files/6ce43cb90658e8cb13177a0210093b567e33e4f6#r198296467) that this was acceptable from an API stability point of view, because no changes in the C API are necessary, only in the C++ one.

Reply:
@piiswrong The method that requires mutability is `GraphExecutor::ReinitGraph`, which can be found [here](https://github.com/mkolod/incubator-mxnet/blob/tensorrt_integration/src/executor/graph_executor.cc#L978:L1078). This doesn't pose any breaking changes to the existing code base, as long as the C API is used to handle frontend bindings.

Reply:
@piiswrong Also, I presume the lines in question are 155-162, not 152. Those are the ones changed by this commit, lines 152-154 and 163-168 are as authored in 2017 by @reminisce.

Reply:
@piiswrong Thoughts about the above?

Reply:
why not pass by value?

Reply:
Also I think it's better to name the functions as InitTensorRT rather than reinitgraph